_config.yml

```diff
@@ -16,8 +16,14 @@ exclude:
   - package.json
   - package-lock.json
   - dash.org-test-web.pem.enc
-paginate: 12
-paginate_path: "/blog/page:num/"
+pagination:
+  enabled: true
+  per_page: 12
+  permalink: '/page:num/'
```

Comment:
Does changing this from `/blog/page:num/` change the behaviour of the site build?